README.md

```diff
@@ -846,4 +846,67 @@ DELAY 500
 STRING :q!
 DELAY 500
 ENTER
-```
\ No newline at end of file
+```
+
+## The DevOps way
+
+Credit: @frank-bee
+
+
+```yaml
+provider "aws" {
+  region = "us-gov-west-1" # AWS GovCloud (US) region
+}
+
+data "aws_ami" "ubuntu" {
+  most_recent = true
+  owners      = ["099720109477"] # Canonical's AWS account ID
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
+  }
+}
+
+resource "aws_instance" "example" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t2.micro"
+  tags = {
+    Name = "VimChecker"
+  }
+
+  provisioner "local-exec" {
```

Comment:
Add destroy time provider or nothing will be killed

Reply:
privisioner should be used only to detect if vim is running. destroy is running in some terminal , some shell